### PR TITLE
103 gdpr implications review submission

### PR DESCRIPTION
**Pull request by Nikomus**
https://trello.com/c/Yp8a8XDH/103-gdpr-implications-review-submission-vvv-ee

### CODE REVIEW

File: src/Components/Checkbox.tsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import styled from 'styled-components'
 
 interface CheckboxProps {
-  label: string
+  label: string | React.ReactNode
```
Comment (edpe) [inline src/Components/Checkbox.tsx:4]: Is React.ReactNode used when the element is a child? I don't fully understand when to use JSX.Element and when to use React.ReactNode.

**Review by edpe — APPROVED** [review]
Looks good :heavy_check_mark:

File: src/Components/Checkbox.tsx (hunk 1 above)
Comment (Nikomus) [reply src/Components/Checkbox.tsx:4]: Looking at this a bit more closely it should probably just be `label: string | React.ReactElement`, as string is a valid ReactNode that type is in the list twice.

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/privacy-policy**](https://my-best-life-j40wywz59-neontribe.vercel.app/privacy-policy) | [**/service/\[id\]**](https://my-best-life-j40wywz59-neontribe.vercel.app/service/[id]) |
|:-:|:-:|
|<a href="https://my-best-life-j40wywz59-neontribe.vercel.app/privacy-policy">Screenshot of /privacy-policy</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/HweuKtcuoQzhpIBqpDcIgaDqCOXJ8SLsWJbJEEytc9klTtYAXQOkj6fERTl8pXmwwsz4mF7A9doFQwLG6LbC1A.png">(view full size)</a> |<a href="https://my-best-life-j40wywz59-neontribe.vercel.app/service/[id]">Screenshot of /service/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/91T2n5XRUecDNkQ8PqKUig_RckGREbu96bRyRnPZaH5xIdeVgZ-envTcUvZ9s4Aa848Ndp0fqHrv3470i6zF3Q.png">(view full size)</a> |







Commit c7e04f57d68c57d875c9de4f71a464bdb7ebc2b1 (https://my-best-life-j40wywz59-neontribe.vercel.app).

**Pull request merged**